packages/fxa-admin-panel/src/components/AccountSearch/index.tsx

```diff
@@ -178,6 +188,7 @@ export const AccountSearch = () => {
 
   const suggestionSelected = (value: string) => {
     setSearchInput(value);
+    setSelectedSuggestion(value);
```

Comment:
nit: afaict the suggestion value is never reset so if someone clicked a suggestion and then later enter the value that's the suggested value, then that search would have autocomplete set to true even though the user typed out the whole value.  very unlikely of course.